pandas/compat/__init__.py

```diff
@@ -365,6 +365,18 @@ def callable(obj):
         return any("__call__" in klass.__dict__ for klass in type(obj).__mro__)
 
 
+if sys.version_info[0:2] < (3, 4):
```

Comment:
prob ok to just do this for PY2. can hyou add a comment and ref to the CPython bug report of what this is doing